app/models/order.rb

```diff
@@ -229,7 +229,7 @@ def finish!(user)
   end
 
   # Sets order.status to 'close' and updates all Ordergroup.account_balances
-  def close!(user)
+  def close!(user, transaction_type)
```

Comment:
This method is used by some (unofficial?) plugins, I'd recommend to revert to the default transaction type when not given.